packages/cloud-functions/src/bigQuery.ts

```diff
@@ -1,9 +1,9 @@
 import { BigQuery } from '@google-cloud/bigquery'
 
 const gcloudProject = process.env.GCLOUD_PROJECT
-export const bigQueryProjectId = 'celo-testnet-production'
-export const bigQueryDataset =
-  gcloudProject === 'celo-mobile-alfajores' ? 'mobile_wallet_dev' : 'mobile_wallet_production'
+export const bigQueryProjectId = gcloudProject
+export const bigQueryExternalDataset = 'external_data_stream' // Dataset used for storing events sourced from external providers
```

Comment:
We're moving the provider-sourced webhook events to a new `external_data_stream` dataset, though we still need to access the `{mobile_wallet_production, mobile_wallet_dev}.app_launched` tables (see below file).